apps/playground/src/app/modules/components/search-bar-module/search-bar-module.component.scss

```diff
@@ -35,6 +35,15 @@
   }
 }
 
+ul {
+  padding-bottom: 12px;
+
+  li {
+    list-style: disc;
+    margin-left: 20px;
```

Comment:
Because of the CSS reset file we use, the bullet point won't be displayed, even with this.

So you actually have to add something like : 
```css
::marker {
    content: unset;
}
```

in the global `styles.scss` file